nanopub/models.py

```diff
@@ -32,7 +31,7 @@ def __init__(self, rdf=None, source_uri=None):
                     f'Expected to find {expected} graph in nanopub rdf, but not found. Graphs found: {list(self._graphs.keys())}.')
 
     @classmethod
-    def from_assertion(cls, assertion_rdf, uri=DEFAULT_URI, introduces_concept=None,
+    def from_assertion(cls, assertion_rdf, uri=DEFAULT_NANOPUB_URI, introduces_concept=None,
```

Comment:
This is the bit I was talking about in the earlier comment - allowing ```uri``` to be set here is very confusing to users as it is actually just a dummy URI. I think I had this option originally because I needed a way to convert triples with local URIs to use the final published URI (usually + some fragment). But now this is done with rdflib blank nodes (assigned a name) so I think that we might in fact be able to get rid of this confusing ```uri``` kwarg entirely.

Reply:
I've created #38